versions.yml

```diff
@@ -6,11 +6,25 @@ unifiedpush_apb: "1.0.1"
 keycloak_apb: "1.1.0"
 metrics_apb: "1.1.0"
 sync_app_apb: "1.1.0"
-js_sdk_app: "2.0.1"
-js_sdk_core: "2.0.1"
-js_sdk_auth: "2.0.0"
-js_sdk_push: "2.0.0"
-js_sdk_security: "2.0.0"
-js_sdk_cordova_plugin_aerogear_metrics: "2.0.0"
-js_sdk_cordova_plugin_aerogear_push: "2.0.0"
-js_sdk_cordova_plugin_aerogear_security: "2.0.0"
+
+# https://github.com/aerogear/aerogear-js-sdk
+js_sdk_app: "2.2.0"
+js_sdk_core: "2.2.0"
+js_sdk_auth: "2.2.0"
+js_sdk_push: "2.2.0"
+js_sdk_security: "2.2.0"
+js_sdk_cordova_plugin_aerogear_metrics: "2.2.0"
+js_sdk_cordova_plugin_aerogear_push: "2.2.0"
+js_sdk_cordova_plugin_aerogear_security: "2.2.0"
+
+# https://github.com/aerogear/ionic-showcase
+ionic-showcase: "0.3.0"
+aerogear/voyager-server-example-task: "0.3.0"
+
+# https://github.com/aerogear/voyager-server
+aerogear/voyager-audit: "0.5.0"
+aerogear/voyager-conflicts: "0.5.0"
+aerogear/voyager-keycloak: "0.5.0"
+aerogear/voyager-metrics: "0.5.0"
+aerogear/voyager-server: "0.5.0"
+aerogear/voyager-tools: "0.5.0"
```

Comment:
We might use underscores instead. e.g.` aerogear_voyager_tools`. But TBH i don't think the aerogear prefix is required anyway.